main/modes/mode_dance.c

```diff
@@ -432,6 +457,50 @@ void danceButtonCb(buttonEvt_t* evt)
     }
 }
 
+void danceTouchCb(touch_event_t* evt)
+{
+    dancePollTouch();
```

Comment:
Is there any feedback during the swipe?

Reply:
Yeah, the screen gets enabled and updated during the entire swipe